quartz/plugins/transformers/ofm.ts

```diff
@@ -260,6 +260,11 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                     value: alias ?? fp,
                   },
                 ],
+                data: {
+                  hProperties: {
+                    class: alias ? "alias" : "",
+                  },
+                },
```

Comment:
you actually don't need this code block any more as all wikilinks get transformed to mdast links which get properly processed by transformers/links.ts